DOCSUP-7099: edit and translate (runningConcurrency, max_parallel_replicas) (#21569)
ClickHouse/ClickHouse: docs/en/operations/settings/settings.md

## max_parallel_replicas {#settings-max_parallel_replicas}

The maximum number of replicas for each shard when executing a query.

Possible values:

-   Positive integer.

Default value: `1`.

**Additional Info** 

> This setting is only useful for replicated tables with a sampling key. A query may be executed faster by executing on several servers in parallel. But query performance may degrade in some cases:

Reviewer: ```suggestion
This setting is only useful for replicated tables with a sampling key. A query may be processed faster if it is executed on several servers in parallel. But query performance may degrade in some cases:
```
